web/packages/teleterm/src/services/vnet/diag.test.ts

```diff
@@ -51,12 +51,33 @@ describe('reportToText', () => {
         ],
       },
     };
+    const sshConfigReport = makeCheckReport({
```

Comment:
Could you add the SSH report to the story for `DocumentVnetDiagReport` at `web/packages/teleterm/src/ui/Vnet/DocumentVnetDiagReport.story.tsx`? It boils down to adding some controls so that it's easier to see the story in different states.

`pnpm storybook` then https://localhost:9002/?path=/story/teleterm-vnet-documentvnetdiagreport--document-vnet-diag-report the controls should be at the bottom, if not there's a button in the top right to show them.

Reply:
yeah that storybook is pretty useful, didn't really know about it. Added in the latest commit.